src/metabase/notification/send.clj

```diff
@@ -388,14 +390,47 @@
   "The default options for sending a notification."
   {:notification/sync? false})
 
+(setting/defsetting notification-cutoff-timestamp
```

Comment:
Given the subtlety of this setting, if we're willing to let it get into master I think it needs a more explicit name.

How about `notification-suppression-cutoff`?

Also, if we're happy with it in master, why not just cut this PR directly to master and merge it into our branch?

Reply:
I was under the impression that we need this for beta deployment first.

Reply:
hmm, why do we want this in master when we don't have a need for it?

Reply:
yea, it might be helpful but getting to master is so much work :(. I just want to make this a quick PR so we can work on others thing.